retrievalmarket/impl/providerstates/provider_fsm.go

```diff
@@ -47,52 +45,46 @@ var ProviderEvents = fsm.Events{
 	// receiving blocks
 	fsm.Event(rm.ProviderEventBlockSent).
 		FromMany(rm.DealStatusOngoing).ToNoChange().
-		From(rm.DealStatusUnsealed).To(rm.DealStatusOngoing).
-		Action(func(deal *rm.ProviderDealState, totalSent uint64) error {
```

Comment:
a `ProviderEventBlockSent` is fired in retrievalmarket/impl/requestvalidation/revalidator.go with an argument; is there any harm in leaving that in, should we remove it just to tidy up?